config/samples/saas_v1alpha1_corsproxy.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: saas.3scale.net/v1alpha1
+kind: CORSProxy
+metadata:
+  name: dev-eng-corsproxy
```

Comment:
CR name should be example like the other controllers?

Reply:
Fixed